### ISSUE TITLE
"Preview on Github" button does nothing

### ISSUE THREAD

**Issue by chrisco23** [Microsoft/vscode#49524]
Yes I know this has been reported elsewhere but those issues were both unhelpful and closed.

- VSCode Version: 1.23.0
- OS Version: Linux arch-desktop 4.16.6-1-ARCH

Steps to Reproduce:

1. Go into Issue Reporter
2. Fill out form and click "Preview on Github"

Does this issue occur when all extensions are disabled?: Yes

**Comment by DickvdBrink** [comment]
What doesn't work? For me it opens a browser tab with the stuff already filled in (and seems to work oke).

**Comment by chrisco23** [comment]
Sorry, meant to say nothing happens (edited).

**Comment by chrisco23** [comment]
Sorry but this seems to not have been a VSCode issue at all but rather had to do with some mime setting issues.

In my particular case, I found that I needed to manually set some mime types for some reason, even though Chromium is my default browser already.

In other cases I was having an issue also with Vivaldi hijacking some browser launching situations, so in case this might help somebody else, I actually found the answer by using dLeon's answer here (except I changed these to "chromium.desktop").  https://forum.vivaldi.net/topic/13636/xdg-open-with-a-url-opens-an-empty-vivaldi-window/2
